local_settings_example.py

```diff
@@ -582,4 +585,21 @@
 
 # }}}
 
+# {{{ cas (optional)
+
+if RELATE_SIGN_IN_BY_CAS_ENABLED:
+    # For these settings, as well as the full list of django-cas-ng settings refer to documentation
+    # at https://github.com/mingchen/django-cas-ng
+    CAS_SERVER_URL = "https://your-cas-login-domain/cas/"
+    CAS_CREATE_USER = True
+    CAS_USERNAME_ATTRIBUTE = 'username'
+
+    # This is an extra addition. To allow one to easily use a more complex attribute mapper
+    # than a simple map in CAS_RENAME_ATTRIBUTES,
+    # we have the cas-config app, which will call the function you define here
+    # with the same parameters as django-cas-ng signal cas_user_authenticated.
```

Comment:
Improve phrasing. Imagine you have no idea what this does and reading this text. :)